src/components/xy_chart/axis/x_axis.js

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { XAxis } from 'react-vis';
+import { EuiXYChartAxisUtils } from '../utils/axis_utils';
+
+const { TITLE_POSITION, ORIENTATION } = EuiXYChartAxisUtils;
+
+export class EuiXAxis extends PureComponent {
+  render() {
+    const {
+      title,
+      titlePosition,
+      orientation,
+      tickSize,
+      tickLabelAngle,
+      tickFormat,
+      tickValues,
+      on0,
+      ...rest
+    } = this.props;
+    return (
+      <XAxis
+        title={title}
+        position={titlePosition}
+        orientation={orientation}
+        on0={on0}
+        tickSize={tickSize}
+        tickFormat={tickFormat}
+        tickValues={tickValues}
+        tickLabelAngle={tickLabelAngle}
+        {...rest}
+      />
+    )
+
+  }
+}
+
+EuiXAxis.displayName = 'EuiXAxis';
+
+EuiXAxis.propTypes = {
+  title: PropTypes.string, /** The axis title */
+  /** The axis title position */
+  titlePosition: PropTypes.oneOf([
+    TITLE_POSITION.START,
+    TITLE_POSITION.MIDDLE,
+    TITLE_POSITION.END,
+  ]),
+  /** The axis orientation */
+  orientation: PropTypes.oneOf([
+    ORIENTATION.TOP,
+    ORIENTATION.BOTTOM,
+  ]),
+  /** Fix the axis at zero value */
+  on0: PropTypes.bool,
+  /** An array of ticks values */
+  ticks: PropTypes.array,
+  /** The height of the ticks in pixels */
+  tickSize: PropTypes.number,
```

Comment:
Again, I dont think this is probably a good prop to have as we are aiming for constancy not flexibility in many cases. @snide would prob be good to ping regarding this